kmir/src/kmir/__main__.py

```diff
@@ -61,6 +62,36 @@ def _kmir_run(opts: RunOpts) -> None:
     print(kmir.kore_to_pretty(result))
 
 
+def _kmir_run_x(opts: RunOpts) -> None:
+    assert opts.file is not None
+    # produce and compile a module and re-load a KMIR object with it
+    smir_info = SMIRInfo.from_file(Path(opts.file))
+
+    # with tempfile.TemporaryDirectory() as work_dir:
+    #     kmir = KMIR.from_kompiled_via_kore(smir_info, symbolic=opts.haskell_backend, target_dir=work_dir)
+    #     result = kmir.run_smir(smir_info, start_symbol=opts.start_symbol, depth=opts.depth)
+    #     print(kmir.kore_to_pretty(result))
+
+    kmir = KMIR.from_kompiled_kore(smir_info, symbolic=opts.haskell_backend)  # leaves out-kore behind
+    result = kmir.run_smir(smir_info, start_symbol=opts.start_symbol, depth=opts.depth)
+    print(kmir.kore_to_pretty(result))
```

Comment:
This would replace the `_kmir_run` implementation but currently the `from_kompiled_kore` is only implemented for symbolic execution, and the legacy backend used by `krun` does not work as expected (the added equations seem to be ignored).

Reply:
The only way to ensure that right now is by generating K instead of KORE, and calling `kompile`.